src/reducers/auth.js

```diff
@@ -0,0 +1,41 @@
+import api from 'utils/api';
+
+const LOGIN_START = 'rmp/users/LOGIN_START';
+const LOGIN_SUCCESS = 'rmp/users/LOGIN_SUCCESS';
+const LOGIN_FAILURE = 'rmp/users/LOGIN_FAILURE';
+
+const initialState = {
+  isLoggedIn: false,
+  isLoggingIn: false,
```

Comment:
maybe better to name this "loading" so it's consistent across other states we'll have?

Reply:
That makes sense while you have only a single `loading` status in a redux module.
If there is more than one, meaningful names help a lot.
I'd go with something like `logInInProgress`.